kolibri/plugins/user/assets/src/app.js

```diff
@@ -12,41 +12,48 @@ const PageNames = require('./state/constants').PageNames;
 
 class UserModule extends KolibriModule {
   ready() {
-    const routes = [
-      {
-        name: PageNames.SIGN_IN,
-        path: '/signin',
-        handler: (toRoute, fromRoute) => {
-          actions.showSignIn(store);
+    coreActions.getCurrentSession(store).then(() => {
+      const routes = [
+        {
+          name: PageNames.ROOT,
+          path: '/',
```

Comment:
I'm assuming the logic for the page switched didn't happen here because you don't have access to the state yet? @christianmemije